plugins/catalog-backend/src/service/DefaultEntitiesCatalog.ts

```diff
@@ -246,21 +248,49 @@ export class DefaultEntitiesCatalog implements EntitiesCatalog {
     // means it'll never try to write down the children again (it assumes that
     // they already exist). This means that without the code below, the database
     // never "heals" from accidental deletes.
-    await this.database<DbRefreshStateRow>('refresh_state')
-      .update({
-        result_hash: 'child-was-deleted',
-        next_update_at: this.database.fn.now(),
```

Comment:
we should still be doing this

Reply:
Will do.

Reply:
I disagree with this one - it's important both here AND in the block above that we set next_update_at, otherwise stitching won't happen as expected and we'll get some nasty bugs

Reply:
Ah, good catch. Will update.